Makefile

```diff
@@ -56,13 +56,28 @@ run-mypy: | build-dev
 	docker run $(dev_args) mypy openslides_backend/ tests/
 
 test: | build-dev
-	docker run $(dev_args) pytest
+	docker run $(dev_args) pytest tests/unit/
 
-run-tests: | build-dev
-	docker run $(dev_args) sh -c "OPENSLIDES_BACKEND_RUN_ALL_TESTS=1 pytest"
 
-run-tests-fast: | build-dev
-	docker run $(dev_args) pytest
+# compose commands
+
+run-dev-compose: | build-dev
+	docker-compose -f docker-compose-dev.yml up -d
+
+run-tests: | run-dev-compose
```

Comment:
This does not stop the compose setup, does it?

This target and `run-tests-cov` should stop it afterwards. For running pytest over and over again you can use `run-compose-bash`.

Reply:
done

Reply:
Correction: If I add it to `make run-tests-cov-no-tty` (the command that gets executed in the GH actions), the first few tests fail for some reason. Leaving it out for now.